src/components/KInput/KInput.vue

```diff
@@ -20,7 +20,7 @@
           :id="inputId"
           :aria-invalid="hasError || charLimitExceeded ? 'true' : undefined"
           class="form-control k-input"
-          :class="`k-input-${size}`"
+          :class="[`k-input-${size}`, $slots['icon'] ? `has-icon icon-${iconPosition}` : '' ]"
```

Comment:
```suggestion
          :class="{ `k-input-${size}`: size, `has-icon icon-${iconPosition}`: $slots['icon'] }"
```
nit

Reply:
I'm afraid this won't work
![Screen Shot 2023-03-10 at 12 14 13 PM](https://user-images.githubusercontent.com/36751160/224380015-32c792b1-d8fc-4fc6-84af-142e06d3bfeb.png)

Reply:
Updated the snippet